libs/sdk-backend-tiger/src/backend/workspace/catalog/availableItemsQueryUtils.ts

```diff
@@ -0,0 +1,47 @@
+// (C) 2021 GoodData Corporation
+import {
+    isMeasure,
+    isArithmeticMeasure,
+    measureLocalId,
+    isSimpleMeasure,
+    isPoPMeasure,
+    isPreviousPeriodMeasure,
+    measureMasterIdentifier,
+    IMeasure,
+} from "@gooddata/sdk-model";
+import { InvariantError } from "ts-invariant";
+
+/*
+ * Availability query measures do not need to contain arithmetic measures and measures derived from them:
+ * they cannot influence the item availability anyway.
+ */
+export function filterMeasuresForAvailabilityQuery(measures: IMeasure[]): IMeasure[] {
```

Comment:
Should we share this with bear? If so, where should it live? sdk-model or sdk-backend-base?